src/devices/Bmxx80/samples/Bme680.sample.cs

```diff
@@ -45,12 +49,21 @@ static void Main()
                         Thread.Sleep(measurementDuration);
 
                         // Print out the measured data
-                        bme680.TryReadTemperature(out var temperature);
-                        bme680.TryReadPressure(out var pressure);
-                        bme680.TryReadHumidity(out var humidity);
+                        bme680.TryReadTemperature(out var tempValue);
+                        bme680.TryReadPressure(out var preValue);
+                        bme680.TryReadHumidity(out var humValue);
                         bme680.TryReadGasResistance(out var gasResistance);
-
-                        Console.WriteLine($"{temperature.Celsius:N2} \u00B0C | {pressure.Hectopascal:N2} hPa | {humidity:N2} %rH | {gasResistance:N2} Ohm");
+                        var altValue = WeatherHelper.CalculateAltitude(preValue, defaultSeaLevelPressure, tempValue);

```

Comment:
does bme680 have a TryReadAltitude as well? If so I believe we should use that instead. If not, then I'm fine with keeping this as is

Reply:
It doesn't. Will keep as is then. I believe #797 will take care of it.